examples/deadline/All-In-AWS-Infrastructure-Basic/python/package/lib/service_tier.py

```diff
@@ -122,6 +126,12 @@ def __init__(self, scope: Construct, stack_id: str, *, props: ServiceTierProps,
             version=props.deadline_version
         )
 
+        secrets_management_settings = {
```

Comment:
I suggest changing this to use the `SecretsManagementProps` interface we provide to be consistent with how we pass props:

```suggestion
        secrets_management_settings = SecretsManagementProps(
            enabled=props.enable_secrets_management
        )
```